pkg/mount/state.go

```diff
@@ -116,6 +116,15 @@ func (s *State) MountOP(what, where, t string, options []string, timeout time.Du
 					MountOption: mountPoint,
 					FstabEntry:  *tmpFstab,
 					Target:      where,
+					PrepareCallback: func() error {
+						// Don't fsck /sysroot as it comes from a loop device and we cant fsck that
+						// image for the loop device is already fsck somewhere else
+						// cos-immutable-rootfs ignored any fsck status, didn't even log it
+						if where != s.Rootdir {
```

Comment:
should probably drop this as we ignore the errors coming and if we mount on RW we want to fsck I guess.